app/services/moves/delete_historic_unissued.rb

```diff
@@ -0,0 +1,11 @@
+module Moves
+  class DeleteHistoricUnissued
+    def self.call
+      new.call
+    end
+
+    def call
+      Move.active.where('date(now()) - date(date) >= 1').in_batches.update_all(deleted_at: Time.now.utc)
```

Comment:
as the [current code is](https://github.com/ministryofjustice/moving-people-safely/blob/42ddd2862d154d835af18f15ffb2eca2289b83d1/app/models/move.rb#L19),  just turns out to be more confusing (at least the naming) IMO.

Also `date(now()) - date(date) >= 1` is kind of a filter. You can have *unissued* moves that are still active, as in, their date has not been reached, so they haven't yet been printed.

If we find a clear name for this, I'm happy to move this to a scope of its own.